README.rst

```diff
@@ -51,6 +52,13 @@ Run the demo:
 
    python cirq-on-iqm/examples/demo_adonis.py
 
+Run code on a real quantum computer:
+
+.. code-block:: bash
+
+   export IQM_SERVER_URL="https://example.com/"
```

Comment:
resolved

Reply:
There's a slash here still :D

Reply:
Also, I'd explicitly say that there can't be a slash because it's an easy thing to e.g. copypaste from somewhere and the user might not understand the resulting error unless it's explicitly mentioned somewhere :)

Reply:
Instead of editing the docs I made the URL / insensitive. Now it doesn't matter whether there is a slash or not :)